docs/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+tocMaxDepth: 2
+---
+
+import { WideTile, Flex } from '@redocly/developer-portal/ui';
+
+# Redocly CLI
+
+Redocly CLI is an open source command-line tool for working with OpenAPI descriptions, developer portals, and other API lifecycle operations including API linting, enhancement, and bundling.
+
+<div>
+  <Flex justifyContent="space-between" flexWrap="wrap">
+    <WideTile to="../redoc/deployment/cli.md" header="Redocly and Redoc">
+      Redoc is our open source API documentation tool. Use `redocly` (this project) to work with your Redoc documentation projects.
+    </WideTile>
+    <WideTile to="./file-management.md" header="Manage OpenAPI descriptions">
```

Comment:
Would it still make sense if we just did "OpenAPI made manageable" here? I'm trying to shorten the titles so they don't wrap!

Reply:
Yeah I tried to stay true to the original and make it look good. It was tough because the original text was not meant for a tile. I like that heading better and will update it.